### PR TITLE
Use new radius request attribute to identify vpn access and modified valid_mac to reject ip address (detect 123.456.789.123 as 12:34:56:78:91:23)

### PR DESCRIPTION
**Pull request by fdurand**
# Description
Added new radius attribute vpn detection for fortigate
Modifed valid_mac that identify some ip address as mac

# Impacts
RADIUS vpn workflow

# Delete branch after merge
YES

# Checklist
- [ ] Document the feature
- [ ] Add unit tests
- [ ] Add acceptance tests (TestLink)

# NEWS file entries

## Enhancements
* Added new radius attribute vpn detection for fortigate
* Fixed valid_mac that identify some ip address as mac

### CODE REVIEW

File: lib/pf/util.pm [hunk 1]
```diff
@@ -296,27 +296,27 @@ our $NON_VALID_MAC_REGEX = qr/^(00|ff)(:\g1){5}$/;
 our $VALID_PF_MAC_REGEX = qr/^[0-9a-f]{2}(:[0-9a-f]{2}){5}$/;
 
 sub valid_mac {
```
Comment (julsemaan) [inline lib/pf/util.pm:302]: since this is such a critical function, @jrouzierinverse, can you take a look at this too just to make sure

Comment (jrouzierinverse) [reply lib/pf/util.pm:302]: This is just a cleanup of whitespace and using a global IP address regex it changes no functionality.

Comment (julsemaan) [reply lib/pf/util.pm:302]: yes, agreed, just wanted to have at least 2 pair of eyes look at it

**Review by jrouzierinverse — COMMENTED** [review]
LGTM

**Comment by fdurand** [comment]
BUMPE !

**Comment by julsemaan** [comment]
Just double checking that this really needs to go in the maintenance

**Comment by fdurand** [comment]
BUMP

**Comment by julsemaan** [comment]
My suggestion: we merge this now and backport to maintenance after 12.1 is released. All good?

**Comment by fdurand** [comment]
i am good with that

**Pull request merged**